### PR TITLE
Add unban support

### PR DESCRIPTION
**Pull request by sim6**

### CODE REVIEW

File: matrix_client/room.py [hunk 1]
```diff
@@ -269,6 +269,21 @@ def ban_user(self, user_id, reason):
         except MatrixRequestError:
             return False
 
+    def unban_user(self, user_id):
+        """ Unban a user from this room
```
Comment (SShrike) [inline matrix_client/room.py]: There's a leading space before "Unban" here, that should probably be removed.

Comment (sim6) [inline matrix_client/room.py]: @SShrike Thanks. Done via 443509a4b1920d4cc56ed5f9714526e7d7678b28.

**Comment by Half-Shot** [comment]
ok to test

**Comment by Half-Shot** [comment]
LGTM (and thanks to @SShrike for catching that). Merging

**Comment by sim6** [comment]
:)

**Pull request merged**